build.xml

```diff
@@ -64,6 +64,7 @@
       <equals arg1="${-cleaned.specification.version}" arg2="14"/>
       <equals arg1="${-cleaned.specification.version}" arg2="15"/>
       <equals arg1="${-cleaned.specification.version}" arg2="16"/>
+      <equals arg1="${-cleaned.specification.version}" arg2="17"/>
```

Comment:
I tried to follow ac754c4bbc5e3e2a988dd266845e8ff3622781b1 as closely as possible.